front-end/src/components/Markdown.tsx

```diff
@@ -87,6 +88,13 @@ export const MarkdownTreeNode: FunctionComponent<{
     return <span className="font-mono">{format(parseTreeNode.child, parameters)}</span>
   }
   if (parseTreeNode.kind === "```") {
+    if (parseTreeNode.language === "table") {
+      return (
+        <div className="my-5">
+          <DrawTable table={parseTreeNode.child} />
+        </div>
+      )
+    }
```

Comment:
Das ist nicht falsch, aber sauberer wäre es, wenn wir hier der normalen Markdown-Syntax für Tabellen folgen:
https://www.markdownguide.org/extended-syntax/#tables
Also ohne, dass man in einen Code-Block rein muss. Dazu muss `parseMarkdown.ts` angepasst werden, die `parseTable`-Funktion sollte auch dort rein.